genie-core/src/main/java/com/netflix/genie/core/jpa/services/JpaFileServiceImpl.java

```diff
@@ -52,7 +52,9 @@ public JpaFileServiceImpl(final JpaFileRepository fileRepository) {
      */
     @Override
 //    @Transactional(propagation = Propagation.REQUIRES_NEW)
-    public void createFileIfNotExists(@NotBlank final String file) throws GenieException {
+    public void createFileIfNotExists(
+        @NotBlank(message = "File path cannot be blank") final String file
```

Comment:
The annotations must match exactly in the interface definitions and the implementations or you will get a runtime error.

Reply:
Yes, bumped into this when testing on my stack.